Week3/homework/2-step3.js

```diff
@@ -1,8 +1,41 @@
 'use strict';
+let range = function (startIndex ,stopIndex) {
+  const values = [];
+  let i = startIndex;
+  while(i <= stopIndex){
+    values.push(i);
+    i++;
+  }
+  return values;
+
+}
+function sayThree(){
+  return "sayThree";
+}
+
+function sayFive(){
+  return "sayFive";
+}
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const values = [];
-  // Add your code here
+
+let newValues = range (startIndex,stopIndex);
+let result = [];
+for (let i = 0; i < newValues.length; i++){
+  if (newValues[i] % 5 === 0 && newValues[i] % 3 === 0){
+       result.push(threeCallback())
+       result.push(fiveCallback())
+  }else if(newValues[i] % 3 === 0){
+    result.push(threeCallback())
+
+  }else if(newValues[i] % 5 === 0){
+    result.push(fiveCallback())
+
+  }
+  
+}
+return result;
 }
 
-threeFive(10, 15, sayThree, sayFive);
+console.log(threeFive(10, 15, sayThree, sayFive).join(", "));
```

Comment:
The output from this line is:

```
sayFive, sayThree, sayThree, sayFive
```

That's not very useful. I would like to be informed which numbers are divisible by 3 and by five.

Overall, your solution is unnecessarily complicated. Please refer to the zip file I posted  in slack for a simple, straight-forward solution.